packages/env/README.md

```diff
@@ -1,38 +1,141 @@
-# `@wordpress/env`
+# `wp-env`
 
-> A zero-config, self contained local WordPress environment for development and testing.
+`wp-env` lets you easily set up a local WordPress environment for building and testing plugins and themes. It's simple to install and requires no configuration.
 
-## Usage
+## Quick (tl;dr) instructions
+
+Ensure that Docker is running, then:
+
+```sh
+$ cd /path/to/a/wordpress/plugin
+$ npx wp-env start
```

Comment:
Here `npx wp-env` is used, but everywhere else it's just `wp-env`. Perhaps a note explaining the difference (local vs global installation) should be included?

Reply:
This section is intended to be an overly brief "tl;dr" section aimed at experienced developers. With that in mind I'd prefer to leave looking up what `npx` is as an exercise for the reader.